src/Molecules/DateRange/DateRange.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import userEvent from '@testing-library/user-event'
+
+import BaseDateRange from './index'
+import { TransProvider } from '../../Atoms/Trans'
+
+const translations = {
+  global: {
+    dateFormat: 'dd/MM/yyyy hh:mm',
+  },
+}

```

Comment:
It will break because the translation component will return the untranslated key (which would be `global.dateFormat`). `date-fns` wouldn't know what to do with the `g` character and throw an error:
```
RangeError: Format string contains an unescaped latin alphabet character `g`
```

The only thing I see is to check if the text was translated or not by doing: 
```js
const format = t('global.dateFormat') === 'global.dateFormat' ? t('global.dateFormat') : 'dd/mm/yyyy'
```

but I don't if it would be better to do it that way, or just set our translations to override the `en` one when defining them.

Reply:
The component could also expect an already translated format.
I don't think the format should be in translations anyway as it is not a text but a format.

If you Export the `MuiPickersUtilsProvider` you can then set the locale globally for the app, you also get back the compatibility with moment, and remove the dependencies on date-fns and the adapter.